elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/SQLQueryEngine.java

```diff
@@ -106,7 +117,7 @@ public Object apply(ResultSet rs) {
 
     @Override
     protected Table constructTable(Class<?> entityClass, EntityDictionary metaDataDictionary) {
-        return new SQLTable(entityClass, metaDataDictionary);
+        return new SQLTable(entityClass, metaDataDictionary, defaultConnectionDetails, connectionDetailsMap);
```

Comment:
The SQLTable shouldn't need the default and connectionDetails map.  It should just be handed the only ConnectionDetail object that pertains to it.

Reply:
ok.